Components/Card/card.css

```diff
@@ -0,0 +1,337 @@
+    --card-image-text-overlay-color: white;
+    --card-primary-button-color: black;
+    --card-primary-button-bg-color: #bae6fd;
+    --card-primary-button-border: none;
+    --card-primary-button-border-radius: 0.6em;
+    --card-button-hover-border-radius: none;
+    --card-secondary-button-color: black; /* i.e. card icon buttons */
+    --card-secondary-button-bg-color: none;
+    --card-secondary-button-border: none;
+    --card-secondary-button-border-radius: 0.6em;
+    --card-button-hover-color: #7dd3fc;
+    --card-icon-size: 1.4rem;
+    --card-badge-bg-color: #bae6fd;
+    --card-badge-border: 1px solid black;
+    --card-badge-border-radius: 0.5em;
+    --card-badge-color: black;
+    --card-button-focus-outline-color: #3b82f6; /* focus color is seen when you use tab key to access buttons or links*/
+}
+
+.card {
+    width: 100%;
+    background-color: var(--card-bg-color);
+    border: var(--card-border);
+    border-radius: var(--card-border-radius);
+    box-shadow: var(--card-box-shadow);
+
+    /* flex - properties */
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    gap: 0.75em;
+}
+
+.card:hover {
+    box-shadow: var(--card-hover-box-shadow);
+}
+
+.card-body {
+    display: flex;
+    gap: 0.8em;
+}
+
+.card-vertical .card-body {
+    flex-direction: column;
+}
+
+.card-horizontal .card-body {
+    flex-direction: row;
+    gap: 0;
+}
+
+.card-image-header-container {
+    position: relative;
+}
+
+.card-header-top .card-image-header-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.card-horizontal .card-image-header-container {
+    display: flex;
+}
+
+.card-with-badge .card-image-header-container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5em;
+}
+
+.card-with-dismiss .card-image-header-container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5em;
+}
+
+.card-horizontal > .card-body > .card-image-header-container > header {
+    margin: 0 auto;
+    margin-top: 0.5rem;
+}
+
+.card-image {
+    width: 100%;
+    border-radius: var(--card-border-radius) var(--card-border-radius) 0 0;
+}
+
+.card-header-top .card-image {
+    border-radius: 0;
+}
+
+.card-only-text .card-image {
+    display: none;
+}
+
+.card-horizontal .card-image {
+    width: 40%;
+    object-fit: cover;
+    border-radius: var(--card-border-radius) 0 0 0;
+}
+
+.card-header-top > .card-body > .card-image-header-container > header {
+    margin-bottom: 0.7em;
+    order: -1;
+}
+
+.card-text-overlay > .card-body > .card-image-header-container > header {
+    position: absolute;
+    bottom: 1em;
+    color: var(--card-image-text-overlay-color);
+}
+
+.card-vertical > .card-body > .card-image-header-container > header {
+    margin-top: 0.5em;
+}
+
+.card-with-badge > .card-body > .card-image-header-container > header {
+    width: 90%;
+    margin: 0 auto;
+}
+
+.card-with-dismiss > .card-body > .card-image-header-container > header {
```

Comment:
Mohit here you should provide header a class and that's it i think no need to do this and same for other header.

Reply:
Yes, I will not merge this branch, would do these changes at the end before merge.

Reply:
Hey, I just added a header class for all, you can check it link below. Thanks! 
[https://github.com/mohitdhatrak/Spark-UI/pull/11/commits/6a3dd81b243589f2c5b449ac5397059921003c0b](url)